lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,115 @@
+        it(`for threat level category ${category}`, async function() {
+          const el = quickRender({ threatLevelCategory: category })!,
+              user = userEvent.setup(),
+              threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true }),
+              categoryTitleCase = category[0].toUpperCase() + category.slice(1);
+
+          await user.hover(threatIndicator);
+          const tooltip = await screen.findByRole('tooltip');
+          expect(tooltip).toHaveTextContent(categoryTitleCase);
+        });
+      });
+    });
   });
 
-  it('should show custom tooltip title', function() {
-    const tooltip = getShallowComponent({ title: 'Extinction Level Threat' }).find(NxToolip);
+  it('should show custom tooltip title', async function() {
+    const el = quickRender({ title: 'Extinction Level Threat' }),
+        user = userEvent.setup(),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true })!;
 
-    expect(tooltip).toHaveProp('title', 'Extinction Level Threat');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Extinction Level Threat');
   });
 
-  it('adds specified class names to the element', function() {
-    const icon = getShallowComponent({ className: 'foo', threatLevelCategory: 'low' }).find(NxFontAwesomeIcon);
+  describe('when presentational prop is set to true', function() {
+    let el: RenderResult;
 
-    expect(icon).toHaveClassName('foo');
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toHaveClassName('nx-threat-indicator--low');
-  });
+    beforeEach(() => {
+      el = quickRender({ presentational: true });
+    });
 
-  it('should hide tooltip when presentational prop is set true', function() {
-    const component = getShallowComponent({ presentational: true });
-    const tooltip = component.find(NxToolip);
-    const icon = component.find(NxFontAwesomeIcon);
+    it('should hide tooltip', async function() {
+      const user = userEvent.setup(),
+          threatIndicator = el.getByRole('presentation', { hidden: true, queryFallbacks: true });
 
-    expect(tooltip).not.toExist();
-    expect(icon).toExist();
-  });
+      expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
+      await user.hover(threatIndicator);
+      expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
+    });
 
-  it('should set the role to "presentation" when presentational prop is true', function() {
-    const icon = getShallowComponent({ presentational: true }).find(NxFontAwesomeIcon);
+    it('should set the role to "presentation"', function() {
+      const threatIndicator = el.getByRole('presentation', { hidden: true, queryFallbacks: true });
 
-    expect(icon).toHaveProp('role', 'presentation');
-  });
+      expect(threatIndicator).toBeInTheDocument();
+    });
+
+    it('should not set the role to "img"', function() {
```

Comment:
Seems redundant with the previous test

Reply:
> Seems redundant with the previous test

a2617d48cca0f385b3faa61fd2981a00885e8093